### DISCUSSION TITLE
Change my nickname is working, but the addr that binding with my twitter is same as the addr in wallet.

### DISCUSSION THREAD

**Discussion by leeduckgo** [DimensionDev/Maskbook#4100] [General]
Change my nickname is working, but the addr that binding with my twitter is same as the addr in wallet.
 「NFT in twitter profile depends on ens in nickname or bio and eth address in bio」.
so…… if I use another nickname(not leeduckgo.eth ), the NFTs should be show normally too?
![image](https://user-images.githubusercontent.com/12784118/130346231-defded17-9770-4ca8-aaa8-1601d63f106c.png)
![image](https://user-images.githubusercontent.com/12784118/130346287-2c203158-c587-43b7-aed6-af3160ed007a.png)

_Originally posted by @leeduckgo in https://github.com/DimensionDev/Maskbook/issues/4055#issuecomment-903226352_

**Comment by AinuX** [comment]
NFTs in twitter profile will not query the current connected wallet address, and there is no binding relationship between persona and wallet, so if you chang your nickname to another one like vitalik.eth, it will list NFTs fetched from vitalik.eth rather than your wallet address.
